site/content/docs/5.0/customize/optimize.md

```diff
@@ -24,10 +24,9 @@ For instance, assuming you're using your own JavaScript bundler like Webpack or
 ```js
 // Import just what we need
 
-// If you're importing tooltips or popovers, be sure to include the Popper.js dependency
+// If you're importing dropdowns, tooltips or popovers, be sure to include the Popper.js dependency
 // import "../../node_modules/popper.js/dist/popper.min.js";
 
-import "../../node_modules/bootstrap/js/dist/util.js";
 import "../../node_modules/bootstrap/js/dist/modal.js";
```

Comment:
it seems are assuming folks use a bundles so the path should be:
```js
import 'bootstrap/js/dist/modal'

Reply:
You're right, at least that's how it should be for Webpack and Parcel. But the documentation for this example also mentions Rollup and I don't know about it, so I assumed the code was written like this for Rollup.

I'll try to get a bundle with Rollup to see how it goes.

Reply:
You're right, at least that's how it should be for Webpack and Parcel. But the documentation for this example also mentions Rollup and I don't know about it, so I assumed the code was written like this for Rollup.

I'll try to get a bundle with Rollup to see how it goes.